src/igprof-navigator

```diff
@@ -184,7 +184,7 @@ def summary(database, out, cumulative):
   if "PATH_INFO" in os.environ and "SCRIPT_NAME" in os.environ:
     absPath = join(os.environ["SCRIPT_NAME"],
                    os.environ["PATH_INFO"].lstrip("/")).rstrip("/")
-    absPath = absPath.replace("/cumulative", "").replace("/self", "").replace("/1", "")
+    absPath = absPath.replace(r"/cumulative", "").replace(r"/self", "").replace(r"/1$", "")
```

Comment:
@gartung , are you sure this works? string.replace do not understand regexp.

Reply:
You are correct. It "worked" because the /1 was not replaced. I pushed the correct change.